app/src/main/java/com/example/triptracker/view/home/HomeUtils.kt

```diff
@@ -95,11 +99,14 @@ fun DisplayItinerary(
               color = md_theme_orange, // This is the orange color
               fontFamily = FontFamily(Font(R.font.montserrat_regular)),
               fontSize = 14.sp)
-          Spacer(modifier = Modifier.height(20.dp))
-          Spacer(modifier = Modifier.heightIn(6.dp).weight(1f))
+          Spacer(modifier = Modifier.height(30.dp).weight(1f))
+          // Spacer(modifier = Modifier.heightIn(6.dp).weight(1f))
```

Comment:
Is this line meant to be uncommented later or just removed?

Reply:
Removed! haha, thanks